cloud-init/copr_test

```diff
@@ -149,7 +149,7 @@ main() {
     name="copr-centos-${uuid%%-*}"
     repo="copr.repo"
 
-    start_container "images:centos/$version" "$name"
+    start_container "images:centos/${version}-Stream" "$name"
```

Comment:
I don't think this works for our current tests, since Centos 7 is still supported (through mid-2024)  and as far as I can tell Centos Stream 7 doesn't exist. Maybe we could make the change: `s/version/centos_version/`, since it looks like our `centos_version` in server-jenkins-jobs matches the lxd image name.

Reply:
Right, thanks for spotting this. Thing is we *also* need the bare CentOS version later in the script, to grab the .rpm from the right COPR location. I think we need to infer when Stream is needed based on the requested version.

Reply:
I would have preferred if the LXD image kept the `centos/N` name for CentOS Stream, after all there's no CentOS non-Streams >= 8, and there's no CentOS Stream 7. I guess they went with the `N-Stream` name because for a period there were both CentOS 8 and CentOS Stream 8.